src/OpenTelemetry.Extensions.Serilog/README.md

```diff
@@ -0,0 +1,29 @@
+# OpenTelemetry.Extensions.Serilog
+
+This project contains a [Serilog](https://github.com/serilog/)
+[sink](https://github.com/serilog/serilog/wiki/Configuration-Basics#sinks) for
+writing log messages to OpenTelemetry.
+
+## Usage Example
```

Comment:
nit: most of our readme files start with the installation of the package itself. we can use same pattern for this one too.

Reply:
Updated. Also added badges.